### PR TITLE
SDKS-1367, SDKS-2260, SDKS-2071 Remove Native SSO

### PR DESCRIPTION
**Pull request by witrisna**
# JIRA Ticket

SDKS-1367, SDKS-2260, SDKS-2071

# Description

Remove Native SSO

# Definition of Done Checklist:

- [ ] Acceptance criteria is met.
- [ ] All tasks listed in the user story have been completed.
- [ ] Coded to standards.
- [ ] Ensure backward compatibility.
- [ ] API reference docs is updated.
- [ ] Unit tests are written.
- [ ] Integration tests are written.
- [ ] e2e tests are written.
- [ ] Functional spec is written/updated.
- [ ] contains example code snippets.
- [ ] Change log updated.
- [ ] Documentation story is created and tracked.
- [ ] Tech debts and remaining tasks are tracked in separated ticket(s).

### CODE REVIEW

**Comment by jeyanthanperiyasamy** [comment]
started reviewing in the middle. i may have few question before i proceed

**Review by rodrigoareis — COMMENTED** [review]
Changes looks good to me. Just one question to be addressed.

File: forgerock-auth/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,33 +1,15 @@
...
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.forgerock.android.auth">
 
-    <uses-permission
-        android:name="android.permission.AUTHENTICATE_ACCOUNTS"
-        android:maxSdkVersion="22" />
-    <uses-permission
-        android:name="android.permission.GET_ACCOUNTS"
-        android:maxSdkVersion="22" />
-    <uses-permission android:name="android.permission.INTERNET" />
```
[the remark this answers is not in the document]
Comment (witrisna) [reply forgerock-auth/src/main/AndroidManifest.xml:11]: Good catch, the internet permission is required.

**Review by spetrov — APPROVED** [review]
LGTM 👍🏻

**Comment by jeyanthanperiyasamy** [comment]
I looked the code, this requires a lot of testing more than reviewing this story...so i told stoyan to create a test plan for this

**Pull request merged**